internal/keys/keys.go

```diff
@@ -77,8 +77,11 @@ func (k *KeyResult) String() string {
 	if k.accountKey != nil {
 		_, _ = fmt.Fprintf(writer, "Signature algorithm \t %s\n", k.accountKey.SigAlgo)
 		_, _ = fmt.Fprintf(writer, "Hash algorithm \t %s\n", k.accountKey.HashAlgo)
-		_, _ = fmt.Fprintf(writer, "Weight \t %d\n", k.accountKey.Weight)
 		_, _ = fmt.Fprintf(writer, "Revoked \t %t\n", k.accountKey.Revoked)
+
+		if k.accountKey.Weight >= 0 { // dont show invalid values
```

Comment:
In which case would this be < 0. Should this also include cases where the weight is more than the max 1000?

Reply:
Just input validation so yeah I could check for max min

Reply:
this is used when decoding a PEM key and there is no weight attached to the key so negative value is used as an empty value so we don't show it